python/tank/platform/software_launcher.py

```diff
@@ -0,0 +1,406 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Defines the base class for DCC application launchers all Tank engines
+should implement.
+"""
+
+import os
+import sys
+import re
+import logging
+import xml.etree.ElementTree as XML_ET
```

Comment:
Is this used? I don't see it being used, and I'm also not in love with the use of the `as XML_ET` in general.